server/src/main/java/org/opensearch/index/remote/RemoteStoreUtils.java

```diff
@@ -91,10 +91,9 @@ static public void verifyNoMultipleWriters(List<String> mdFiles, Function<String
                     && (!nodesByPrimaryTermAndGen.get(nodeIdByPrimaryTermAndGen.v1()).equals(nodeIdByPrimaryTermAndGen.v2()))) {
                     throw new IllegalStateException(
                         "Multiple metadata files from different nodes"
+                            + "having same primary term and generations "
                             + nodeIdByPrimaryTermAndGen.v1()
```

Comment:
Should we log the other node ID as well?

Reply:
We just have the hashes of node id here , so it won't help much .